docs/content/introduction/what-is-porter.md

```diff
@@ -0,0 +1,12 @@
+---
+title: What is Porter?
+description: Detailed look at how Porter does its magic 🎩✨
+---
+
+Porter is an implementation of the [Cloud Native Application Bundle](https://cnab.io/) specification and creates installers, known as bundles, that understand how to install not only your application but its infrastructure and configuration. A bundle helps you package up the logic for installing your application so that you can hand it off to another team, a customer, or just a co-worker who doesn’t know all the ins and outs of your application and provide them a consistent installation experience that doesn’t require them to know what tools you use to deploy or how the sausage is made.
```

Comment:
Reading this sentence makes me really not like the `s/invocation images/bundle installers/` I just did - if we can come up with a better name I'm down tho